code/game/objects/items/weapons/tanks/tank_types.dm

```diff
@@ -140,6 +140,8 @@
 	item_state = "plasmaman_tank_belt"
 	slot_flags = SLOT_BELT
 	force = 5
+	volume = 3

```

Comment:
This is going to be a major problem.

normally these tanks have a volume of `70`. You're reducing it to `3`. Even if you min-max the amount of distribution pressure, they're still going to run out of plasma in about ~15-25 minutes.

I'd recommend a tank volume of about ~25-30. It'll be enough for it to last for about 3-4 hours, which is more than enough for the vast vast majority of shifts, but isn't as big as the standard tank volume of `70`.